Source/Xamarin/Prism.Ninject.Forms/Extensions/DependencyServiceBindingResolver.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Ninject.Planning.Bindings.Resolvers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Ninject.Activation;
+using Ninject.Infrastructure;
+using Ninject.Planning.Bindings;
+using Ninject.Components;
+
+using Binding = Ninject.Planning.Bindings.Binding;
+
+namespace Prism.Ninject.Extensions
+{
+    /// <summary>
+    /// Resolves missing dependencies using <see cref="Xamarin.Forms.DependencyService"/>
+    /// </summary>
+    public class DependencyServiceBindingResolver : NinjectComponent, IMissingBindingResolver
```

Comment:
Handles resolving from DependencyService when Ninject does not find the binding.